lib/course_planner_web/templates/email/attendance_missing.html.eex

```diff
@@ -0,0 +1,5 @@
+<div>
```

Comment:
Did you check how this look like? I am not sure you have those variable available, for example `@teacher_names` and `@offered_course_name`

Reply:
hmm god point. no i haven't :P
this PR was on WIP and it seems i missed this part of it completely.
sorry will fix it now

Reply:
Done. there is now a test for this